docs/docs/post-css.md

```diff
@@ -2,7 +2,53 @@
 title: PostCSS
 ---
 
-This is a stub. Help our community expand it.
+PostCSS transforms extended syntaxes and features into modern, browser-friendly CSS. This guide will show you how to get started with Gatsby and PostCSS.
 
-Please use the [Gatsby Style Guide](/docs/gatsby-style-guide/) to ensure your
-pull request gets accepted.
+### Installation and Configuration
```

Comment:
I would add a very short note here like in the `Sourcing from Wordpress` doc that explains that from this point, the guide assumes that a Gatsby site is initialized: https://www.gatsbyjs.org/docs/sourcing-from-wordpress/#quick-start